### PR TITLE
Remove compatibility hack

### PR DESCRIPTION
**Pull request by amoj**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,9 @@
   "homepage": "https://github.com/Sleavely/exodus-migrations",
   "repository": "github:Sleavely/exodus-migrations",
   "license": "MIT",
+  "engines" : {
+    "node" : ">=10.10.0"
+  },
```
Comment (Sleavely) [inline package.json:22]: I really like that you considered this.

File: src/utils/fs.spec.js [hunk 2]
```diff
@@ -151,6 +151,20 @@ describe('findUpwardsFile()', () => {
...
 describe('listDirectoryFiles()', () => {
-  it.todo('lists files in the supplied directory')
+  it('lists files in the supplied directory', async () => {
+    const { listDirectoryFiles } = jest.requireActual('./fs')
+    const { Dirent, constants } = jest.requireActual('fs')
+
+    const { UV_DIRENT_DIR, UV_DIRENT_FILE } = constants
+
+    const file = new Dirent('file', UV_DIRENT_FILE)
+    const directory = new Dirent('directory', UV_DIRENT_DIR)
```
[the remark this answers is not in the document]
Comment (amoj) [reply src/utils/fs.spec.js:161]: Yup, it will make tests cleaner. Will update =)

File: src/utils/fs.spec.js [hunk 3]
```diff
@@ -151,6 +151,29 @@ describe('findUpwardsFile()', () => {
...
+    const files = await listDirectoryFiles()
+
+    expect(files).toMatchObject([ file.name ])
+  })
+  it('ignores subdirectories in the supplied directory', async () => {
+    const { listDirectoryFiles } = jest.requireActual('./fs')
+    const { Dirent, constants } = jest.requireActual('fs')
+    const { UV_DIRENT_DIR } = constants
+
+    const directory = new Dirent('directory', UV_DIRENT_DIR)
```
Comment (Sleavely) [inline src/utils/fs.spec.js:172]: I'm trying to read up on the constructor arguments for Dirent but I can't seem to find anything about it. Also, `UV_DIRENT_DIR` seems to be an undocumented constant; where'd you find out about this pattern? 🗺️ ❔

Comment (Sleavely) [reply src/utils/fs.spec.js:172]: Found the same usage pattern in Node's internal tests for fs, but it required some digging to get there 😄 
https://github.com/nodejs/node/blob/c17dcb32533aa007dfbf507d22c28ef3c7c11c29/test/parallel/test-fs-readdir-types.js#L114

Still curious on how you found about it 😮

Comment (amoj) [reply src/utils/fs.spec.js:172]: Just went through the code in the nodejs github repo. =)

**Pull request merged**